bootstrap/kubeadm/config/webhook/webhookcainjection_patch.yaml

```diff
@@ -1,13 +1,13 @@
 # This patch add annotation to admission webhook config and
 # the variables $(CERTIFICATE_NAMESPACE) and $(CERTIFICATE_NAME) will be substituted by kustomize.
-apiVersion: admissionregistration.k8s.io/v1
-kind: MutatingWebhookConfiguration
-metadata:
-  name: mutating-webhook-configuration
-  annotations:
-    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+# apiVersion: admissionregistration.k8s.io/v1beta1
+# kind: MutatingWebhookConfiguration
+# metadata:
+#   name: mutating-webhook-configuration
+#   annotations:
+#     cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
 ---
-apiVersion: admissionregistration.k8s.io/v1
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
Any reason to change it to v1beta1?

Reply:
It's a mismatch between the existing manifests, the webhooks get generated as v1beta1 due to the issue I mentioned in your backport controller-runtime PR. The kustomize build won't work because the GVK's mismatch. It was broken in master because we didn't have any webhooks.

Reply:
We need to upgrade CR because otherwise `make generate` reverts it back to v1beta1 and breaks kustomize

Reply:
Got it, thanks!